textproc/recode/Portfile

```diff
@@ -47,6 +57,6 @@ post-destroot {
 test.run            yes
 test.target         check
 test.env            CC=${configure.cc} \
-                    DYLD_LIBRARY_PATH=${worksrcpath}/src/.libs \
-                    PATH=${frameworks_dir}/Python.framework/Versions/2.7/bin:$env(PATH)
-test.args           PYTHON=${prefix}/bin/python2.7
+    DYLD_LIBRARY_PATH=${worksrcpath}/src/.libs \
+    PATH=${python.prefix}/bin:$env(PATH)
```

Comment:
You are certain that changing this from args to env is correct? Tests still work in trace mode and find the right things?

Reply:
I have only updated the variables without moving anything. Previous version fail tests.
Now works fine without trace mode. In trace mode I have the following error.
```
Making check in tests
/Applications/Xcode.app/Contents/Developer/usr/bin/make  check-local
rm -f Recode.c && \
	/opt/local/Library/Frameworks/Python.framework/Versions/3.7/bin/cython -3 -o Recode.body.c ./Recode.pyx && \
	echo '#include "config.h"' > Recode.c && \
	cat Recode.body.c >> Recode.c
Traceback (most recent call last):
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.7/bin/cython", line 6, in <module>
    from pkg_resources import load_entry_point
ImportError: cannot import name 'load_entry_point' from 'pkg_resources' (unknown location)
make[2]: *** [Recode.c] Error 1
make[1]: *** [check-am] Error 2
make: *** [check-recursive] Error 1
Command failed:  cd "/opt/local/var/macports/build/_Users_rainbow_sources_github_macports-ports_textproc_recode/recode/work/recode-3.7.6" && /usr/bin/make check PYTHON=/opt/local/Library/Frameworks/Python.framework/Versions/3.7/bin/python3.7 CYTHON=/opt/local/Library/Frameworks/Python.framework/Versions/3.7/bin/cython
Exit code: 2
Warning: The following existing files were hidden from the build system by trace mode:
  /opt/local/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/pkg_resources/__init__.py
  /private/var/select/sh
The following file would have been hidden from the build system by trace mode if it existed:
  /usr/gnu/include
Error: Failed to test recode: command execution failed
Error: See /opt/local/var/macports/logs/_Users_rainbow_sources_github_macports-ports_textproc_recode/recode/main.log for details.
Error: Follow https://guide.macports.org/#project.tickets to report a bug.
Error: Processing of port recode failed
```

Reply:
that failure in trace mode is because ```py-cython``` should declare a runtime dependency on ```py-setuptools``` but doesn't; fixed in 379c7bb (switched to ```depends_lib```).

Reply:
thank you @reneeotten !